index-functions-cpp.xml

```diff
@@ -1704,6 +1717,14 @@
         <function name="from_time_t"/>
     </class>
 
+    <class name="std::chrono::sys_time" link="cpp/chrono/system_clock" since="c++20">
```

Comment:
Note that `sys_time` is actually an alias of `std::chrono::time_point<system_clock, Duration>`. I'm marking it as a class for convenience.

Similarly, `utc_time`, `tai_time`, `gps_time`, and `file_time` are all alias templates of `time_point`.

Reply:
Shouldn't it point to time_point description then? Also, wouldn't `<typedef />` tag be a better option in this case or does it not fit well here?

Reply:
Currently https://en.cppreference.com/w/cpp/chrono/time_point does not mention these aliases. it would be confusing if these entries point there.

There are overloads (`operator<<`, `from_stream`) for these types, that are not otherwise defined for `time_point`.